test/Benchmarks/src/Text/Regex_Bench.enso

```diff
@@ -0,0 +1,84 @@
+from Standard.Base import all
+
+import Standard.Base.Data.Text.Regex.Internal.Replacer.Replacer
+
+from Standard.Test import Bench, Faker
+
+
+type Data
+    Value ~two_nums ~just_two_nums ~four_nums ~a_regex ~regexes ~replacers ~one_num_regex ~two_nums_regex
+
+    create =
+        faker = Faker.new
+        a_regex = Regex.compile "(\d)ab(\d)"
+        Data.Value (build_two_nums faker) (build_just_two_nums faker) (build_four_nums faker) a_regex (build_regexes faker) (build_replacers faker) (Regex.compile "(\d+)") (Regex.compile "(\d+)[a-z]+(\d+)")
```

Comment:
Could this line be split into multiple ones? As

- since #11374 it should be possible to:

>Each argument in a type-constructor definition may be specified on its own (indented) line.

Reply:
This is a constructor invocation, not a definition -- should that also work? When I moved the arguments to multiple lines, I got syntax errors.

Reply:
```ruby
from Standard.Base import all

type X
   Y
      a
      b

main = X.Y
    <| 1 
    <| 2
```